patches/ember-source@5.12.0.patch

```diff
@@ -0,0 +1,87 @@
+diff --git a/dist/ember-template-compiler.js b/dist/ember-template-compiler.js
+index 01999af0c4c125af3c713318db7eb85fa492f72d..3edcfd7ca66cfd31d6acde6eff84dcce8d4f0866 100644
+--- a/dist/ember-template-compiler.js
++++ b/dist/ember-template-compiler.js
+@@ -9485,10 +9485,10 @@ var define, require;
+                   return Object.defineProperty(node, "escaped", {
+                     enumerable: !1,
+                     get() {
+-                      return deprecate$1("The escaped property on mustache nodes is deprecated, use trusting instead"), !this.trusting;
++                      return !this.trusting;
+                     },
+                     set(value) {
+-                      deprecate$1("The escaped property on mustache nodes is deprecated, use trusting instead"), this.trusting = !value;
++                      this.trusting = !value;
```

Comment:
Any luck finding what was triggering this message? Ideally we'd fix it there, rather than silencing the message.

Reply:
Yep. It looks fairly benign.

The escaped property on Mustache Nodes is deprecated and trusting, which equals not escaped, should be used instead.

The problem is that even Ember is not following its best practices yet on this version. 🤣 

Glimmer was still using the escaped property

For example, on `@glimmer+syntax@0.84.3`:

```
  MustacheStatement(mustache) {
    if (this.handledByOverride(mustache)) {
      return;
    }

    this.buffer += mustache.escaped ? '{{' : '{{{';

    if (mustache.strip.open) {
      this.buffer += '~';
    }

    this.Expression(mustache.path);
    this.Params(mustache.params);
    this.Hash(mustache.hash);

    if (mustache.strip.close) {
      this.buffer += '~';
    }

    this.buffer += mustache.escaped ? '}}' : '}}}';
  }
```

It's easier to simply silent this message for now. It will probably go away on later versions anyway.

Reply:
> The problem is that even Ember is not following its best practices yet on this version. 🤣

🤦 

Ok, patching sounds good then! Maybe we just add a comment linking to the relevant part of glimmer/syntax, so the reason is obvious to anyone else that works on it in future?